### ISSUE TITLE
[false positive] rtrim can be used instead.

### ISSUE THREAD

**Issue by stephanedelprat** [kalessil/phpinspectionsea#627]
Using version 2.3.11.1

Code : 

```php
$new = preg_replace('! +$!m', '', $string);
```

Triggers a message : 

> 'rtrim($string, ' ')' can be used instead. less... (Ctrl+F1)
> Analyzes regular expressions and reports possible improvements.

It's not correct, because of the **m** modifier in the regex.

**Comment by kalessil** [comment]
Good catch @stephanedelprat, I'll fix the case.

**Comment by kalessil** [comment]
Fixed.
